### DISCUSSION TITLE
Can't render character using escape code and hex value

### DISCUSSION THREAD

**Discussion by YTN0** [mikke89/RmlUi#352] [Q&A]
I am trying to display a character (specifically a font image from font awesome) using the html escape sequence. Specifically, something along the lines of <button>&#xF04B;</button>

However, this does not seem to render the image as expected. It displays the actual code itself instead of the glyph. I tried many variations of this per the HTML recommendations (https://www.w3.org/International/questions/qa-escapes), and nothing seems to work.

I did look through the docs, and can't see mention of how this should be handled.

Any help would be appreciated.

Thanks!

**Comment by mikke89** [comment]
We don't currently support numeric escape codes. However, you can just write your symbols directly into your RML documents, just make sure everything is encoded in UTF-8 as that is always assumed in the library. In C++ you can use the `u8` string prefix.

For reference, here are the supported named characters: [Documentation - Text elements](https://mikke89.github.io/RmlUiDoc/pages/cpp_manual/text_elements.html#html-characters).

By the way, for questions about the library, please submit these as a new discussion in the future instead of an issue. Thanks!
